python/ray/tune/integration/docker.py

```diff
@@ -13,6 +14,9 @@ class DockerSyncer(NodeSyncer):
     without a custom sync client. The sync client defaults to
     ``DockerSyncClient`` instead.
 
+    Set the env var `TUNE_SYNCER_VERBOSITY` to increase verbosity
+    of syncing operations.
```

Comment:
Maybe give a hint which values indicate what? I.e. 0 = no output, 999 = full output. Is there anything between?

Reply:
(Isn't this actually 0 vs 1 anyway?)